semantic_conventions/trace/database.yaml

```
@@ -402,6 +402,94 @@ groups:
           value MUST NOT be set.
         examples: ['public.users', 'customers']
 
+  - id: db.cosmosdb
+    type: span
+    extends: db
+    prefix: db.cosmosdb
+    brief: >
+      Call-level attributes for CosmosDb.
+    attributes:
+      - id: client_id
+        type: string
+        brief: Unique cosmos client instance id.
+        examples: '3ba4827d-4422-483f-b59f-85b74211c11d'
+      - id: operation_type
+        type:
+          allow_custom_values: false
+          members:
+            - id: Invalid
+              value: 'Invalid'
+            - id: Create
+              value: 'Create'
+            - id: Patch
+              value: 'Patch'
+            - id: Read
+              value: 'Read'
+            - id: ReadFeed
+              value: 'ReadFeed'
+            - id: Delete
+              value: 'Delete'
+            - id: Replace
+              value: 'Replace'
+            - id: Execute
+              value: 'Execute'
+            - id: Query
+              value: 'Query'
+            - id: Head
+              value: 'Head'
+            - id: HeadFeed
+              value: 'HeadFeed'
+            - id: Upsert
+              value: 'Upsert'
+            - id: Batch
+              value: 'Batch'
+            - id: QueryPlan
+              value: 'QueryPlan'
+            - id: ExecuteJavaScript
+              value: 'ExecuteJavaScript'
+        brief: Cosmsodb Operation Type.
+        requirement_level: required
+      - ref: user_agent.cosmosdb
```

```suggestion
      - ref: user_agent.original
```

why cosmosdb suffix?